### ISSUE TITLE
Remove button does not trigger "change" event on "recordfinder" field

### ISSUE THREAD

**Issue by termiyanc** [octobercms/october#3591]
Hello.

I have "Product" model and two fields in my form:

```
discount:
    label: Discount
    type: recordfinder
    span: left
```
```
discounted_price:
    label: Discounted price
    type: number
    dependsOn: discount
    trigger:
        action: empty
        field: discount
        condition: value[]
    disabled: true
    span: right
```

As you can see, i would like to clear "discounted_price" field when "discount" field gets empty. But this does not work for me when i press remove button at "discount" field and confirm the removing, value of "discounted_price" field remains the same.

But if i manually triggers "change" event on "discount" field after removing

```
jQuery('#Form-field-Product-discount').trigger('change')
```

then trigger works and value of "discounted_price" gets empty.

So i conclude that remove button does not trigger "change" event on "recordfinder" field.

My versions are (from `composer list`):

```
laravel/framework                     v5.5.39            The Laravel Framework.
...
october/backend                       v1.0.434           Backend module for October CMS
october/cms                           v1.0.434           CMS module for October CMS
october/rain                          v1.0.434           October Rain Library
october/system                        v1.0.434           System module for October CMS
```

**Comment by termiyanc** [comment]
@LukeTowers Maybe later. As you can already see)

**Comment by LukeTowers** [comment]
@termiyanc as I can already see what?

**Comment by termiyanc** [comment]
@LukeTowers as you can see, that i am not so speedy about this question here)

**Comment by bennothommo** [comment]
Closing as it has been over a month since any activity on this occurred and we are trying to figure out what issues are still relevant. If this is still something that you would like to see through to fruition please respond and we can get the ball rolling.

**Comment by danielbidala** [comment]
This issue hasn't solved yet! I can confirm it exists. Please reopen!

One of my field dependsOn a recordfinder field and when I clear the value from recordfinder, change event is not triggered.

**Comment by LukeTowers** [comment]
@danielbidala feel free to make a PR to fix this, I think you can probably add 
`data-request-success="$('#<?= $field->getId() ?>').trigger('change')"` 
to https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/recordfinder/partials/_recordfinder.htm#L31

Try it out locally and then if it works make a PR and I'll merge it.

**Comment by danielbidala** [comment]
@LukeTowers Thanks for the suggestion I tested your solution and it solved the problem. I tried to make a PR but I think I've been doing it in the wrong way and I closed it. Sorry I'm new on GitHub.

**Comment by Samuell1** [comment]
@danielbidala you did it good but you need to make PR to develop branch.

**Comment by bennothommo** [comment]
Fixed by 5862683a7a97e6bdf8833a1dc7cfe50876776e47.
